authentik/core/api/authenticated_sessions.py

```diff
@@ -115,3 +123,14 @@ class AuthenticatedSessionViewSet(
     filterset_fields = ["user__username", "session__last_ip", "session__last_user_agent"]
     ordering = ["user__username"]
     owner_field = "user"
+
+    @action(detail=False, methods=["POST"], pagination_class=None, filter_backends=[])
+    def bulk_delete(self, request: Request) -> Response:
+        """Bulk revoke all sessions for multiple users"""
+        serializer = BulkDeleteSessionSerializer(data=request.data)
+        serializer.is_valid(raise_exception=True)
+
+        user_ids = serializer.validated_data.get("user_ids", [])
+        deleted_count, _ = AuthenticatedSession.objects.filter(user_id__in=user_ids).delete()
```

Comment:
```suggestion
        user_pks = body.validated_data.get("user_pks", [])
        deleted_count, _ = AuthenticatedSession.objects.filter(user_pk__in=user_pks).delete()
```
With the other changes